README.md

```diff
@@ -56,6 +61,23 @@ Avaliable endpoints are:
 
 Currently only the latest error type is recorded for a stream. However all error types are being logged.
 
+The `HLSMonitorService` can also be controlled through code:
+
+```typescript
+import { HLSMonitorService } from "@eyevinn/hls-monitor";
+
+// initialize a new instance of HLSMonitorService
+const hlsMonitorService = new HLSMonitorService();
+
+// create a new hls-monitor
+const streams = ["streams-to-monitor/manifest.m3u8"];
+hlsMonitorService.getMonitor(streams);
```

Comment:
I think it would be a bit more clear with:
```
hlsMonitorService.monitor.create(streams)
```

where `monitor` is a get function in the HLSMonitorService class:
```
get monitor() {
  if (!this.monitor) {
    this.monitor = new HLSMonitor([]);
  }
  return this.monitor
}
```

Reply:
I've updated to use the logic that you suggested. I agree that it is clearer.